recipes/icu/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "56.1" %}
+{% set version_under = version.replace(".", "_") %}
+
+package:
+  name: icu
+  version: {{ version }}
+
+source:
+  fn:  icu4c-{{ version_under }}-src.tgz
+  url: http://download.icu-project.org/files/icu4c/{{ version }}/icu4c-{{ version_under }}-src.tgz
+  md5: c4a2d71ff56aec5ebfab2a3f059be99d
+  patches:
+    # include an extra header when using mingw
+    - icu4c-4_9_1-mingw-w64-mkdir-compatibility.patch  # [win]
+    # omit lib prefix on libraries
+    - icu-config.patch                                 # [win]
+
+build:
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python
```

Comment:
I am changing it to Windows only in this PR ( https://github.com/conda-forge/icu-feedstock/pull/3 ).